static/admin/config.yml

```
@@ -23,11 +23,11 @@ collections:
         widget: "list"
         required: false
         fields:
-          - { label: "Title", name: "blogTitle", widget: "string"}
+          - { label: "Title", name: "blogTitle", widget: "string", pattern: ['.{5,60}', "Must have 5-60 characters"]}
           - { label: "Publish Date", name: "publishDate", widget: "datetime" }
-          - { label: "Label", name: "blogLabel", widget: "string" }
-          - { label: "Description", name: "blogDescription", widget: "string" }
-          - { label: "Body", name: "blogBody", widget: "markdown" }
+          - { label: "Label", name: "blogLabel", widget: "string", pattern: ['.{3,20}', "Must have 3-20 characters"]} }
```

```suggestion
          - { label: "Label", name: "blogLabel", widget: "string", pattern: ['.{3,20}', "Must have 3-20 characters"]}
```
please remove the additional bracket, or the project will not compile 🗒️

---

fixed